Sources/StreamChat/Models/Member.swift

```diff
@@ -136,10 +149,7 @@ public enum MemberRole: String, Codable, Hashable {
         case "owner":
             self = .owner
```

Comment:
This is probably nit. IMHO either keep all the default values in the initial switch or let there be only the asignment in the `default` branch...

Reply:
All the default values are in the initial switch, and only the assignment is in the `default` branch